packages/react-examples/src/react-theme-provider/ThemeProvider/docs/ThemeProviderOverview.md

```diff
@@ -0,0 +1,3 @@
+`ThemeProvider` applies contextual theme to its child components. Themes can be applied to your entire app, to specific subtrees, or to individual components. You can also infinitely nest and override themes. By default, Fluent UI React components are applied with Fluent theme.
```

Comment:
To hopefully reduce confusion for non-devs who land on this page, it's probably good to clarify that this is just a utility which doesn't render anything by itself. Here's my attempt at that.
```suggestion
`ThemeProvider` is a utility that applies contextual theming to its child components. Themes can be applied to your entire app, to specific subtrees, or to individual components. You can also infinitely nest and override themes. By default, Fluent UI React components use a Fluent theme.
```